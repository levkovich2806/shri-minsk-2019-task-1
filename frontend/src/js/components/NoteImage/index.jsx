import React, { Component } from "react";
import styles from "./index.module.scss";

class NoteImage extends Component {
  render() {
    const { url } = this.props;
    return (
      <div className={styles.image}>
        <img src={url} alt={url} />
      </div>
    );
  }
}
export default NoteImage;
