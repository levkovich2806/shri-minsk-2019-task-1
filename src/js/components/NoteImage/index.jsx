import React, { Component } from "react";
import styles from "./index.module.scss";

class NoteImage extends Component {
  render() {
    const { url } = this.props;
    return (
      <div className={styles.image}>
        <img src={url} />
      </div>
    );
  }
}
export default NoteImage;
