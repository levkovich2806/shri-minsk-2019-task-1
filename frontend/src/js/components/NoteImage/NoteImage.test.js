import React from "react";
import NoteImage from "./index.jsx";
import renderer from "react-test-renderer";

describe("NoteImage", function () {
  test("NoteImage rendering", () => {

    const component = renderer.create(<NoteImage url={'http://test.ru/logo.png'} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
