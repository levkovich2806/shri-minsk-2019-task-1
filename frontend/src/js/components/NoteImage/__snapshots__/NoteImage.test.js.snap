// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoteImage NoteImage rendering 1`] = `
<div
  className="image"
>
  <img
    alt="http://test.ru/logo.png"
    src="http://test.ru/logo.png"
  />
</div>
`;
